it_hardware/data/ir_model_fields.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="new_date_lot_serial_x_warranty_date" model="ir.model.fields">
+        <field name="name">x_warranty_date</field>
+        <field name="model_id" ref="stock.model_stock_lot"/>
+        <field name="ttype">date</field>
+        <field name="field_description">Warranty Date</field>
+    </record>
+    <record id="new_date_helpdesk_ti_x_warranty_date" model="ir.model.fields">
+        <field name="name">x_warranty_date</field>
+        <field name="model_id" ref="helpdesk.model_helpdesk_ticket"/>
+        <field name="ttype">date</field>
+        <field name="field_description">Warranty Date</field>
+        <field name="related">lot_id.x_warranty_date</field>
+    </record>
+    <record id="new_field_to_set_warranty_date" model="ir.model.fields">
+        <field name="name">x_warranty_days</field>
+        <field name="model_id" ref="product.model_product_template"/>
+        <field name="field_description">Warranty Days</field>
+        <field name="ttype">integer</field>
+    </record>
```

Comment:
Good! Could you therefore add values to some of the products? And adapt the knowledge where it needs to? Thanks!

Reply:
Did you update the knowledge already?